logging/src/main/java/org/jboss/as/logging/formatters/StructuredFormatterResourceDefinition.java

```diff
@@ -349,10 +349,11 @@ private static String modelValueToMetaData(final ModelNode metaData) {
 
     private static class AddStructuredFormatterStepHandler extends LoggingOperations.LoggingAddOperationStepHandler {
         private final Class<? extends StructuredFormatter> type;
+        private final AttributeDefinition[] attributes;
 
         private AddStructuredFormatterStepHandler(final Class<? extends StructuredFormatter> type, final AttributeDefinition[] attributes) {
-            super(attributes);
             this.type = type;
+            this.attributes = attributes;
```

Comment:
Cannot these attributes be retrieved from the context.getRegistration instead in the performRuntime method?

Reply:
I originally kept this in order to keep the changes minimal, but yes, with some more work this can be eliminated.
I've updated this PR with more extensive changes required handle this removal.